controls/cis_rhel7.yml

```diff
@@ -2490,7 +2499,7 @@ controls:
     - auditd_data_retention_action_mail_acct
     - var_auditd_action_mail_acct=root
     - auditd_data_retention_admin_space_left_action
-    - var_auditd_admin_space_left_action=halt
+    - var_auditd_admin_space_left_action=cis_rhel7
```

Comment:
Unfortunately, the rule `auditd_data_retention_admin_space_left_action` isn't able to accept a multiple value variable. I believe this is the reason of the CI fail, the rule service_auditd_enabled fails because auditd can't start because its configuration is broken because the remediation of this rule inserts both values there which is an invalid config.  The OVAL, Bash and Ansible need to be changed to handle this. We have done a similar change in `auditd_data_retention_space_left_action` , so you can do the same thing here